config.json

```diff
@@ -2028,8 +2052,8 @@
       {
         "groupId": "com.android.installreferrer",
         "artifactId": "installreferrer",
-        "version": "1.1.2",
-        "nugetVersion": "1.1.2.1",
+        "version": "2.0",
```

Comment:
This one is frozen as it changed from Apache to Android SDK license and the template will have to be updated to not ship the Java binary code.  Seems like nothing else in this PR requires a new version here so we can revert this.

As an aside, I just looked at documenting why certain artifacts are frozen yesterday, so we can track these better going forward: https://github.com/xamarin/AndroidX/pull/893.

Reply:
I bumped this one, because it seems like it is needed for Google Analytics v4

https://github.com/xamarin/GooglePlayServicesComponents/issues/685

I know - I had to dig that one out and info is scattered across 2 repos.

Reply:
done